scripts/codegen_driverwrappers/generate_driver_wrappers.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+import os
+import jinja2
+
+def render(tpl_path):
+    path, filename = os.path.split(tpl_path)
+    return jinja2.Environment(
+        loader=jinja2.FileSystemLoader(path or './')
+    ).get_template(filename).render()
+
+n = len(sys.argv)
+if ( n != 3 ):
+    sys.exit("The template file name and output file name are expected as arguments")
+# set template file name, output file name
+driver_wrapper_template_filename = sys.argv[1]
+driver_wrapper_output_filename = sys.argv[2]
+
+# render the template
+result = render(driver_wrapper_template_filename)
+
+# write output to file
+outFile = open(driver_wrapper_output_filename,"w")
+outFile.write(result)
```

Comment:
Complaint when building with `clang -std=c99 -pedantic -Wall -Wextra` (included on our CI with `tests/scripts/all.sh test_clang_opt`):
```
psa_crypto_driver_wrappers.c:2283:34: error: no newline at end of file [-Werror,-Wnewline-eof]
#endif /* MBEDTLS_PSA_CRYPTO_C */
```
Looks like jinja is stripping off the newline at the end of the last line?

Reply:
https://github.com/ARMmbed/mbedtls/commit/f7a054bc07fb96ca9a8ebf5c6a64b6cde1cbd86c

Reply:
Thanks for fixing this before I got around to !